src/Controls/GlobeControls.js

```diff
@@ -911,8 +932,8 @@ function GlobeControls(view, targetCoordinate, range, globeRadius, options = {})
         this.camera.layers.enable(layerTHREEjs);
     }
 
-    _handlerMouseMove = onMouseMove.bind(this);
-    _handlerMouseUp = onMouseUp.bind(this);
+    _handlerMouseMove = _onMouseMoveListener;
```

Comment:
Could you remove `_handlerMouseMove` and `_handlerMouseUp` and could you use `let _onMouseMoveListener` and `let _onMouseUpListener`  instead?

Reply:
I pushed a new commit one the same branch (its the first time I use PR so I hope that is the right way to do). I hope the modifications are what you expect.

Just one precision: I moved up the two `_onMouseMoveListener` and `_onMouseUpListener` (stayed as `const`) variable definitions just before the `onMouseDown` in order to respect the ESLint rule `no-use-before-define`.

Reply:
This is good, that's what I wanted.